vassal-app/src/main/java/VASSAL/configure/ConfigureTree.java

```diff
@@ -2676,6 +2676,7 @@ public boolean importData(TransferHandler.TransferSupport support) {
 
           if (remove(getParent(sourceNode), cutObj)) {
             insert(target, convertedCutObj, childIndex);
+            postPasteFixups(convertedCutObj);
```

Comment:
HA, This is the actual fix. I don't completely missed that in the other code paths!

Reply:
In the "drag" version, this was (on the one hand) "fixing" the problem, but it was creating a new "bad data" message in the console -- in other words, it WASN'T the thing making the cut/paste thing work. Cut/Paste already didn't generate the problem and so didn't get the extra warning message. So this DOES belong here (for other reasons) but DOESN'T really fix this problem.